examples/02_classification.ipynb

```diff
@@ -550,17 +589,12 @@
     }
    },
    "source": [
-    "sktime offers two other ways of solving multivariate time series classification \n",
-    "problems:\n",
+    "`sktime` offers two other ways of solving multivariate time series classification problems:\n",
```

Comment:
"solving multivariate time series problems" to "building estimators for multivariate time series problems"